src/order/rel_classes.lean

```diff
@@ -84,13 +84,22 @@ instance is_total.to_is_refl (r) [is_total α r] : is_refl α r :=
 lemma ne_of_irrefl {r} [is_irrefl α r] : ∀ {x y : α}, r x y → x ≠ y | _ _ h rfl := irrefl _ h
 lemma ne_of_irrefl' {r} [is_irrefl α r] : ∀ {x y : α}, r x y → y ≠ x | _ _ h rfl := irrefl _ h
 
-lemma not_rel (r) [is_irrefl α r] [subsingleton α] (x y) : ¬ r x y :=
+lemma not_rel_of_subsingleton (r) [is_irrefl α r] [subsingleton α] (x y) : ¬ r x y :=
 subsingleton.elim x y ▸ irrefl x
 
+lemma not_rel_iff_of_subsingleton (r) [is_irrefl α r] [subsingleton α] (x y) : r x y ↔ false :=
```

Comment:
`subsingleton.rel_iff_false`? Why do you need this lemma at all? `simp` can use `¬ r x y` to replace `r x y` by `false`.

Reply:
Oh, I wasn't aware of that. I'll get rid of this.